### DISCUSSION TITLE
Query su lista fatture

### DISCUSSION THREAD

**Discussion by infabula77** [fattureincloud/api#216] [Q&A] [answered]
Salve 

ho creato uno script PHP molto semplice che estrae le fatture per anno e crea una lista,
il sistema ha funzionato fino ad oggi.

Oggi ho notato che però funziona per l'anno corrente, in pratica se cerco le fatture del 2022 mi compare una pagina

"Service Unavailable
The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later."

La variabile anno viene presa dall'url con un GET e inserita in una query "date>='2022-01-01' and date<='2022-12-31'"
sull'api reference mi funziona.

Qualcuno ha idea di cosa possa aver sbagliato?
Non sono un grande programmatore :-(

Grazie

**Comment by emajo** [comment, marked as answer]
Ciao @infabula77 ti chiedo di scrivere al nostro supporto clienti per andare ad indagare più approfonditamente la causa di questo problema che non abbiamo mai riscontrato.

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiorno @infabula77 sei riuscito a risolvere tramite supporto clienti?
